### PR TITLE
Add static links

### PR DESCRIPTION
**Pull request by evans-g-crsj**

### CODE REVIEW

**Review by francisl — APPROVED** [review]
👍

File: src/pages/variantEntity/TabFrequencies.tsx [hunk 1]
```diff
@@ -145,46 +159,44 @@ const makeRowFromFrequencies = (frequencies: Frequencies) => {
 
   return [
     {
-      cohortName: 'TopMed',
+      cohort: ['TopMed', `https://bravo.sph.umich.edu/freeze8/hg38/variant/snv/${locus}`],
```
Comment (jecos) [inline src/pages/variantEntity/TabFrequencies.tsx:162]: should be an object?

File: src/pages/variantEntity/TabFrequencies.tsx [hunk 2]
```diff
@@ -109,10 +109,24 @@ const internalColumns = (
...
+const INDEX_COHORT_NAME = 0;
+const INDEX_COHORT_LINK = 1;
+
 const externalColumns = [
   {
     title: 'Cohort',
-    dataIndex: 'cohortName',
+    dataIndex: 'cohort',
+    render: (cohort: string) => {
+      const cohortName = cohort[INDEX_COHORT_NAME];
```
Comment (jecos) [inline src/pages/variantEntity/TabFrequencies.tsx:120]: should we have an object?

Comment (evans-g-crsj) [reply src/pages/variantEntity/TabFrequencies.tsx]: good point

**Pull request merged**